kiwmi/luak/kiwmi_keyboard.c

```diff
@@ -16,6 +16,29 @@
 #include "input/keyboard.h"
 #include "luak/kiwmi_lua_callback.h"
 
+static int
+l_kiwmi_keyboard_configure(lua_State *L)
+{
+    struct kiwmi_keyboard *keyboard =
+        *(struct kiwmi_keyboard **)luaL_checkudata(L, 1, "kiwmi_keyboard");
+
+    struct xkb_rule_names rules = {
+        .rules   = luaL_checkstring(L, 2),
+        .model   = luaL_checkstring(L, 3),
+        .layout  = luaL_checkstring(L, 4),
+        .variant = luaL_checkstring(L, 5),
+        .options = luaL_checkstring(L, 6),
+    };
+    wlr_log(WLR_INFO, " %s ", lua_tostring(L, 2));
+    struct xkb_context *context = xkb_context_new(XKB_CONTEXT_NO_FLAGS);
+    struct xkb_keymap *keymap =
+        xkb_map_new_from_names(context, &rules, XKB_KEYMAP_COMPILE_NO_FLAGS);
```

Comment:
I could only find `xkb_keymap_new_from_names`, which seems to be the same, is this an alias? What do flags do, why are they hardcoded to 0?

Reply:
oh yeah that is an alias I renamed it to match the function directly rather then the define